_data/api-swagger.yaml

```diff
@@ -649,6 +650,51 @@ definitions:
         }
       }
       ```
+
+  Itsme Identification:
+    allOf:
+    - $ref: '#/definitions/Verification'
+    - type: object
+      properties:
+        PhoneNumber:
+          type: string
+          readOnly: true
+          description:             
+            "The mobile phone number of the signer.
+            Must conform to E.164,
+            [the international public telecommunication numbering plan](https://en.wikipedia.org/wiki/E.164),
+            which requires the country calling code (Only the Belgian country calling code is supported: +32)."
+          example: "+32123456789"
+        Attributes:
+          type: map of string
+          readOnly: true
+          description: |
+            Contains all available Itsme Identification attributes.
+            These attributes may change, therefore we cannot guarantee the availability of any of these attributes.
```

Comment:
Maybe put in a little bit: '..may change by the **(Itsme) vendor**, therefore we ..' to indicate we do not have control over the itsme attributes?

Reply:
Verifications which return attributes as well, like idin or eidas, have the same explanation. If this explanation is to be changed I think it should be changed for all of these verifications so they stay uniform.